code/src/App.js

```diff
@@ -1,9 +1,88 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+
+import { API_URL, API_LIKES_URL } from './utilities/Api'
+
+import ThoughtForm from 'components/ThoughtForm'
+import ThoughtList from './components/ThoughtList'
 
 export const App = () => {
+
+  const [thoughtList, setThoughtList] = useState([])
+  const [loading, setLoading] = useState(false)
+  const [newThought, setNewThought] = useState('')
+  const [counter, setCounter] = useState(0)
```

Comment:
Nice that you gathered all these declarations and did not repeat yourself everywhere like I did  😄